app/backend/templates/onboarding2.md

```diff
@@ -27,5 +27,5 @@ We would really appreciate it if you could add a photo and fill in at least the
 
 Thank you so much!
 
-Emily from Couchers.org
+Emily from Couchers.org  
```

Comment:
revert trailing whitespace change.

Reply:
It's ugly markdown stuff to get a newline

Reply:
[confirmed.](https://spec.commonmark.org/0.29/#hard-line-breaks)